pandas/io/gbq.py

```diff
@@ -62,6 +54,16 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         Service account private key in JSON format. Can be file path
         or string contents. This is useful for remote server
         authentication (eg. Jupyter/IPython notebook on remote host).
+    auth_local_webserver : boolean, default False
+        Use the `local webserver flow`_ instead of the `console flow`_
+        when getting user credentials.
+
+        .. _local webserver flow:
```

Comment:
i think these break linting, need a ``#noqa`` I think

Reply:
Adding `# noqa` breaks the link. Sphinx generates a link like `pandas/doc/build/html/generated_single/pandas.read_gbq.html#noqahttp://google-auth-oauthlib.readthedocs.io/en/latest/reference/google_auth_oauthlib.flow.html#google_auth_oauthlib.flow.InstalledAppFlow.run_local_server` when I add this.

I do not get any lint errors when I run `git diff upstream/master -u -- "*.py" | flake8 --diff` and `scripts/validate_docstrings.py pandas.read_gbq` also passes.